test/v2/login.js

```diff
@@ -14,7 +15,7 @@ module.exports = {
 
   adminAuth: asyncWrap(async function (test) {
 
-    test.expect(2)
+    test.expect(2)  // eslint-disable-line no-magic-numbers
```

Comment:
You're needing this enough and have a good reason I suggest disabling it at the file level